README.md

```diff
@@ -1,26 +1,74 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+# hpc-libs
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+[![hpc-libs tests](https://github.com/charmed-hpc/hpc-libs/actions/workflows/ci.yaml/badge.svg)](https://github.com/charmed-hpc/hpc-libs/actions/workflows/ci.yaml)
+[![Release](https://github.com/charmed-hpc/hpc-libs/actions/workflows/release-libs.yaml/badge.svg)](https://github.com/charmed-hpc/hpc-libs/actions/workflows/release-libs.yaml)
+![GitHub License](https://img.shields.io/github/license/charmed-hpc/hpc-libs)
+[![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#hpc:ubuntu.com)
 
-Use links instead.
--->
+A collection of charm libraries for authoring HPC charms 📖🖋️
 
-# hpc-libs
+The hpc-libs charm libraries are used within the [Juju](https://juju.is) charms that compose
+Charmed HPC. They are standalone libraries, and should be managed
+as charm libraries, with installation via `charmcraft fetch-lib ...`, after which they may be
+imported and used as normal Python modules. The current charm libraries in hpc-libs include:
+
+* `is_container` - a library for detecting the virtualization environment the charm is running within.
+* `slurm_ops` - a library for managing Slurm cluster operations via snap or systemd.
+
+## ✨ Getting Started
+
+Each charm library contains documentation and usage information in its module-level docstring.
+Pretty documentation, along with installation instructions, can be viewed on Charmhub:
+
+* [`is_container` documentation](https://charmhub.io/hpc-libs/libraries/is_container)
+* [`slurm_ops` documentation](https://charmhub.io/hpc-libs/libraries/slurm_ops)
+
+## 🤔 What's next?
+
+If you want to learn more about all the things you can do with the hpc-libs charm libraries,
+or have any further questions on what you can do with the charm libraries, here are some
+further resources for you to explore:
+
+* [Open an issue](https://github.com/charmed-hpc/slurmutils/issues/new?title=ISSUE+TITLE&body=*Please+describe+your+issue*)
```

Comment:
```suggestion
* [Open an issue](https://github.com/charmed-hpc/hpc-libs/issues/new?title=ISSUE+TITLE&body=*Please+describe+your+issue*)
```

Reply:
:clown_face: alright, we should be good now